sql/updates/characters/2015_08_26_00_characters_335.sql

```diff
@@ -0,0 +1,16 @@
+-- Keep only the highest guid PvE or PvP (not bones) corpse per player guid
+DELETE c FROM `corpse` c LEFT JOIN
+(
+    SELECT MAX(`corpseGuid`) AS id
```

Comment:
Fails on a fresh DB and rev: https://github.com/TrinityCore/TrinityCore/commit/e68ff4186e685de00362b12bc0b5084a4d6065dd as there's no corpseGuid.